README.md

```diff
@@ -1,16 +1,12 @@
 # infrastructure
 
-Repo to track code + issues relating to IPFS infrastructure for users + devs.
-This includes:
+This repository contains the technical infrastructure of the IPFS community.
 
-Computing:
-- Gateway nodes
-- Bootstrap nodes
-- CI servers (jenkins, travis, etc)
+- Gateway (http://gateway.ipfs.io) and bootstrap nodes, in `solarnet/`
+- Seeding of $important objects, in `seeding/`
 
-Communications:
-- Github repos
-- IRC Channel
-- Mailing Lists
```

Comment:
>  it'd be about "using the infrastructure", while ipfs/infrastructure.git would be about "running the infrastructure".

Yeah this is the distinction i was going for too. i'm +1 to moving the listing over for users to consume. still want one listing somewhere that whoever will be taking care of infra can know where everything is + make sure it's running / accounted for. (e.g. i dont expect freenode or google groups or https://botbot.me/freenode/ipfs/ to fail at all, but should still be accounted for somewhere in case someone needs to replace a piece).